### ISSUE TITLE
Remove the tooltip in combination with the icon-warning icon-request-reading-room

### ISSUE THREAD

**Issue by christinach** [pulibrary/orangelight#3907]
### Steps to recreate
Go to [search results Mudd](https://catalog.princeton.edu/?f%5Blocation%5D%5B%5D=Mudd+Manuscript+Library) 

- Use the mouse and hover over the yellow triangle icon warning.  A tooltip is triggered with extra information 'Items at this location must be requested'.
- Use the keyboard and tabkey. The icon is not focusable. The same information will not be provided to the tab key user.
- Use the screenreader the icon has an aria-hidden attribute even though it provides information to the sighted users who can use a mouse. 
- When you hover over the icon the tooltip adds an aria-describedby which is useless since it is aria-hidden.

### Issues
[Understanding SC 2.1.1](https://www.w3.org/WAI/WCAG22/Understanding/keyboard)

Pointer actions should have a keyboard equivalent 
The tooltip is adding an aria-describedby in the icon span which is useless since it has an aria-hidden true.
Availability can be misleading for different users since the icon tooltip is only read by some users.

### WCAG Conformance level (A, AA, AAA)
A

### User impact
All users don't have the same experience and do not get the same information.

### Recommended fix
Remove the icon warning icon reading room

### Acceptance criteria
- [x] I can successfully use this feature with Voice Over in Safari.
- [x] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...

### Notes
part of #3878
